### PR TITLE
Social tab added with twitter and GitHub buttons

### PR DESCRIPTION
**Pull request by Sehbaz**
Please review the new tab for the social buttons

### CODE REVIEW

File: src/_layouts/layout.njk [hunk 1]
```diff
@@ -18,6 +18,7 @@ title: LeicesterJS
     <meta name="theme-color" content="#2e353f" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="stylesheet" href="/css/styles.css" />
+
```
[the remark this answers is not in the document]
Comment (Sehbaz) [reply src/_layouts/layout.njk]: hello, you mean this feature should be removed ?

[the remark this answers is not in the document]
Comment (kmdev1) [reply src/_layouts/layout.njk]: Hey mate - So I may be being stupid haha, however it looks like line 21 was added but is just a blank line and has no feature :)

[the remark this answers is not in the document]
Comment (Sehbaz) [reply src/_layouts/layout.njk]: Aha! , got it I have update the file and removed the empty space.
I hope this will sort things out.

**Comment by Sehbaz** [comment]
is everything ok now ?

**Comment by kmdev1** [comment]
Perfect buddy - sorry to have been picky haha :P

**Comment by Sehbaz** [comment]
but still waiting to be merged.

**Comment by kmdev1** [comment]
@joshghent 😉

**Pull request closed without merging**